code/game/machinery/hologram.dm

```diff
@@ -250,6 +252,11 @@ Possible to do for anyone motivated enough:
 		record_stop()
 
 /obj/machinery/holopad/attackby(obj/item/P, mob/user, params)
+	
+	if(is_wire_tool(P) && panel_open)
+		wires.interact(user)
+		return
+
```

Comment:
I'm not sure this will be needed when the other two suggested changes are set. Make sure to give it a test without these lines.

Reply:
did the other two changes, doesn't work without that line